From 1775f8e4c5e2f66853f6e6f202fdd32706eb2d95 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Wed, 12 Jan 2011 21:18:21 +0100
Subject: [PATCH] Revert "config: HTML file generation: use the installed copy of xorg.css"

xorg.css is shipped in Debian through xorg-sgml-doctools, which is
probably not a package we want to make all our packages depend on.

So, as an interim measure, keep the local copy of that file.

This reverts commit 8cbca8a10761d1ea75a75bafa647632d6c0dac71.
---
 specs/.gitignore  |    1 +
 specs/xmlrules.in |    9 +++++----
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/specs/.gitignore b/specs/.gitignore
index 6fa5c31..09b6a89 100644
--- a/specs/.gitignore
+++ b/specs/.gitignore
@@ -2,3 +2,4 @@
 *.ps
 *.pdf
 *.txt
+*.css
diff --git a/specs/xmlrules.in b/specs/xmlrules.in
index 7687860..2af4130 100644
--- a/specs/xmlrules.in
+++ b/specs/xmlrules.in
@@ -37,10 +37,11 @@ spec_DATA += $(doc_sources:.xml=.txt)
 endif
 
 if HAVE_STYLESHEETS
-XMLTO_FLAGS += 			\
-	-m $(XSL_STYLESHEET)	\
-	--stringparam img.src.path=$(abs_builddir)/	\
-	--stringparam html.stylesheet=$(STYLESHEET_SRCDIR)/xorg.css
+XMLTO_FLAGS += -m $(XSL_STYLESHEET) --stringparam img.src.path=$(abs_builddir)/
+
+spec_DATA += xorg.css
+xorg.css: $(STYLESHEET_SRCDIR)/xorg.css
+	$(AM_V_GEN)cp -pf $(STYLESHEET_SRCDIR)/xorg.css $@
 endif
 
 CLEANFILES += $(spec_DATA)
-- 
1.7.2.3

